runbotics/runbotics-scheduler/src/scheduler-database/credential-attribute/credential-attribute.entity.ts

```diff
@@ -0,0 +1,41 @@
+import { PrimaryColumn, Entity, Column, OneToOne, JoinColumn, OneToMany, ManyToOne, PrimaryGeneratedColumn, CreateDateColumn, UpdateDateColumn, ManyToMany, Unique } from 'typeorm';
+import { Secret } from '#/scheduler-database/secret/secret.entity';
+import { Credential } from '#/scheduler-database/credential/credential.entity';
+import { Tenant } from '../tenant/tenant.entity';
+
+@Entity({ schema: 'scheduler' })
+@Unique(['name', 'credentialId'])
+export class CredentialAttribute {
+    @PrimaryGeneratedColumn('uuid')
+    id: string;
+
+    @Column({ name: 'name'  })
+    name: string;
+
+    @ManyToOne(() => Tenant)
+    @JoinColumn({ name: 'tenant_id' })
+    tenant: Tenant;
+
+    @Column('uuid', { name: 'tenant_id' })
+    tenantId: string;
```

Comment:
Do we need it here? We have it in credential and CredentialAttributes[] sits there, so I think it will be enough?